app/components/case-studies-components/CaseStudiesTitleText.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+
+interface CaseStudiesTitleTextProps {
+  wordOne: string;
+  wordTwo: string;
+  wordThree: string;
+}
+
+const CaseStudiesTitleText: React.FC<CaseStudiesTitleTextProps> = ({
+  wordOne,
+  wordTwo,
+  wordThree,
+}) => {
+  return (
+    <div className="flex flex-col lg:flex-row">
+      <div className="flex">
+        <p className="z-20  text-4xl font-bold text-black200 dark:text-white900 md:text-[3rem]">
+          {wordOne}
+        </p>
+        <div className="relative ml-2 flex w-fit flex-col self-center md:ml-3 lg:hidden">
+          <p className="z-20 text-4xl font-bold text-black200 dark:text-white900 md:text-[3rem]">
+            {wordTwo}
+          </p>
+          <div className="absolute z-10 h-[1rem] w-full translate-y-[1.6rem] bg-orange-200 md:h-[1.358rem] md:translate-y-[1.7rem]" />
+        </div>
+      </div>
+      <div className="relative ml-4 flex w-fit flex-col self-center md:mt-4 lg:hidden">
```

Comment:
Since you have the same style and structure, repeated multiple times, you could benefit from breaking out the repeated parts into their own sub-component or function. I believe we touched on this, but I don't see the changes.

Reply:
I have refactored this whole section. I think it's a lot more readable and reusable now